src/migration/mark-staking-events-as-processed-migration.js

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable no-await-in-loop */
+import BaseMigration from './base-migration.js';
+
+class MarkStakingEventsAsProcessedMigration extends BaseMigration {
+    constructor(migrationName, logger, config, repositoryModuleManager, blockchainModuleManager) {
+        super(migrationName, logger, config);
+        this.repositoryModuleManager = repositoryModuleManager;
+        this.blockchainModuleManager = blockchainModuleManager;
+    }
+
+    async executeMigration() {
+        this.logger.info('Marking old blockchain events as processed');
+        for (const blockchain of this.blockchainModuleManager.getImplementationNames()) {
+            const timestamp = Date.now();
+            const block = await this.blockchainModuleManager.getLatestBlock(blockchain);
+            const query = `update blockchain
+                           set last_checked_block     = ${block.number},
+                               last_checked_timestamp = ${timestamp}
+                           where blockchain_id = 'otp:2043'`;
```

Comment:
This still needs to be handled.
Might remove it, documentation says WHERE clause is optional
https://dev.mysql.com/doc/refman/8.0/en/update.html

Reply:
I agree, but this is one-time migration that is already executed, so we don't need to remove it now.